src/main/java/edu/kit/kastel/sdq/artemis4j/grading/git/VCSTokenCloningStrategy.java

```diff
@@ -53,11 +55,12 @@ public void performClone(String repositoryUrl, CloneCommand command, ArtemisConn
 
     private String generateNewToken(ArtemisConnection connection) throws ArtemisNetworkException {
         log.info("Generating new VCS access token");
-        var assessor = connection.getAssessor();
-        // The max expiration date is (today + 1 year), see https://github.com/ls1intum/Artemis/blob/921b1884fb1a3028512e61023f254f1365fec14b/src/main/java/de/tum/in/www1/artemis/web/rest/AccountResource.java#L175
+        connection.getAssessor();
```

Comment:
This looks like a mistake? Shouldn't this be `var assessor  = connection.getAssessor()`?

Reply:
Assessor is not used until re-assigned